react-native-purchases-ui/src/index.tsx

```diff
@@ -65,12 +70,19 @@ export default class RevenueCatUI {
    */
   public static PAYWALL_RESULT = PAYWALL_RESULT;
 
-  public static presentPaywall(params: PresentPaywallParams = {}): Promise<PAYWALL_RESULT> {
-    return RNPaywalls.presentPaywall(params);
+  public static presentPaywall({
+                                 offering,
+                                 displayCloseButton = true
```

Comment:
Nitpick but is it possible to extract the default to a constant? Like `private const PRESENT_PAYWALL_DEFAULT_DISPLAY_CLOSE_BUTTON` (maybe too long...), so we don't have to repeat it.

Reply:
Yeah I think it's because this didn't exist initially on iOS, so we had to keep the default the same.
We can have a different default in RN, let's just make sure we document it well here in code and in the docs.